check_attribute-style_access.py

```diff
@@ -0,0 +1,49 @@
+import ast
+import os
+import sys
+
+class DataFrameAttributeVisitor(ast.NodeVisitor):
```

Comment:
What is the goal of this class?

Reply:
Sorry, it got mistakenly commited, I will remove it. It was a script to try to automatically find out occurences of attribute-style accesses but it was not working out well..